gui/src/app/components/expert-contact-card/expert-contact-card.js

```diff
@@ -24,6 +24,8 @@ class ExpertContactCard extends React.Component {
   */
   _showContact() {
     this.setState({showContact: true});
+    this.props.analytics.addCategory(this).addCategory('Show Contact Details').sendEvent(
```

Comment:
Did you mean to nest addCategory here?

Reply:
This would result in something like `Store > Entity Details > ExpertContactCard > Show Contact Details`. Chaining the `addCategory` calls adds a new section to that list.